### PR TITLE
DOCS-220 fixing nits #time 20m

### PR DESCRIPTION
**Pull request by ptimmons**

### CODE REVIEW

File: docs/release_notes_128t_5.2.md [hunk 1]
```diff
@@ -27,17 +27,17 @@ Alternatively, refer to the **[List of Releases](about_releases.md)** page for r
...
 ------
-- **I95-37459 Show Commands for Services:** The [show fib](cli_reference.md#show-fib) and [show fib lookup](cli_reference.md#show-fib-lookup)commands have been enhanced to provide more granular path-related debugging.
+- **I95-37459 Show Commands for Services:** The [show fib](cli_reference.md#show-fib) and [show fib lookup](cli_reference.md#show-fib-lookup) commands have been enhanced to provide more granular path-related debugging.
 ------
-- **I95-37510 [AppID Modules](concepts_appid.md#appid-using-modules) can be run as systemd unit:** The AppId manager now can handle the module execution as systemd unit instead of subprocess, in addition to the existing script-based method. Use REST APIs to propagate module registrations and results with AppId manager. 
+- **I95-37510 [AppID Modules](concepts_appid.md#appid-using-modules) can be run as systemd units:** Support for module execution as a systemd unit in addition to the existing script-based method, and the use of REST APIs to propagate module registrations and results.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_128t_5.2.md:32]: Added support for module execution...

File: docs/release_notes_128t_5.2.md [hunk 2]
```diff
@@ -66,5 +66,5 @@ Alternatively, refer to the **[List of Releases](about_releases.md)** page for r
 
 ## Caveats
 
-- **I95-26627 Prevent static route interface next hops with the same global id:** If two interfaces with the same global id are an HA pair, only one should be permitted as a static route next hop. If the configuration is committed and one of the interfaces is deleted, FRR will delete the entire static route. 
+- **I95-26627 Prevent static route interface next hops with the same global id:** in HA configurations with a shared interface, only one node in the pair should be configured as a static route next hop. Othrewise, deleting the shared interface from one node will also cause the static route to be deleted.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_128t_5.2.md:69]: ...with the same global **ID: In** HA configurations...

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.2.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.2.md:32]: edit made.

File: docs/release_notes_128t_5.2.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.2.md:69]: edit made.

**Pull request merged**